libraries/AC_PrecLand/AC_PrecLand.cpp

```diff
@@ -231,33 +272,33 @@ void AC_PrecLand::handle_msg(mavlink_message_t* msg)
 
 void AC_PrecLand::run_estimator(float rangefinder_alt_m, bool rangefinder_alt_valid)
 {
-    const struct inertial_data_frame_s& inertial_data_delayed = _inertial_history.front();
+    const struct inertial_data_frame_s *inertial_data_delayed = (*_inertial_history)[0];
```

Comment:
Leaving this as a reference by adding another `*` would save a lot of code churn and dereferencing!